### ISSUE TITLE
Unable to play certain games

### ISSUE THREAD

**Issue by Infinity6542** [Jigsaw-Code/outline-server#1627]
I've noticed that certain games such as Roblox and War Thunder (😢) can't be played when connected to my Outline server. It works with other VPNs (namely Hotspot Shield).

It might be worth noting that I'm running Outline on an ARM server (OCI), so I'm using [this script](https://github.com/seia-soto/outline-server-multiarch).

I can't seem to get the error code for most of the games (will edit this if I get more), but the one for Roblox is `Error code 279. ID = 17 `. Minecraft seems to work fine. I've opened the correct ports for some games on my Oracle instance, but that didn't help. My server is set up to use the port 443 by default.

Edit: I thought this was simply to do with the default port at first, but I haven't seen mentions of this anywhere else and [this page](https://www.getoutlinevpn.com/en/blog/fast-gaming-experience-with-outline-vpn) suggests that what I'm having isn't normal. Correct me if I'm wrong though.

**Comment by Infinity6542** [comment]
Yes, I've checked that the 443 port is opened for TCP and UDP. I also opened the required ports for the games, though I'm pretty sure that shouldn't matter?

**Comment by sbruens** [comment]
This might require more work to investigate the root cause of the issue. If you are able to provide more information (e.g. server logs, client logs, game logs, traffic logs) it could help us to troubleshoot this issue.

**Comment by Infinity6542** [comment]
The only information that I have are in the original post. I'd be happy to collect more if you can provide instructions or tell me what information to collect.

**Comment by sbruens** [comment]
I was able to successfully play games on Roblox while connected to an Outline server on my Android phone. This suggests that there may be something specific to your machine or VPS setup that is causing the issue. Can you connect to Roblox from within the server at all?

Here are some things to look into:
 
- **Check the server logs:** Run `docker logs shadowbox` from inside the server to get the Outline server logs. You can also try running the server with environment variable `LOG_LEVEL=debug` set so it prints more debug logs in Docker.

- **Capture network traffic:** On the client side, you can use a tool like Wireshark to capture network traffic while trying to connect to one of those sites. This can be helpful to see if there are any network-related issues preventing the connection.

- **Check for IP restrictions:** It is possible that the sites you're having issue with are blocking or restricting access from OCI, the location of your server, or even your specific IP to prevent abuse. You could try a different provider or region to see if that resolves it.

- **Review your firewalls:** Make sure that you have correct firewall rules at both the machine level (your VPS) and the provider level (your VPS provider's infrastructure). It sounds like you have the right ports open, but it's good to check at both levels. I'm not familiar with how OCI does this exactly.

Finally, note that you are running an unofficial Outline server image; we don't currently support ARM (https://github.com/Jigsaw-Code/outline-server/issues/1009). You could give the server core https://github.com/Jigsaw-Code/outline-ss-server a try on your machine as that should work on ARM. It doesn't come with the management API; instead it uses a config file to set access keys. But maybe that works for you, in which case it's probably something to do with the unofficial server image you're using.

**Comment by Infinity6542** [comment]
I just cleared iptables and it seems to be working on my side now. Will have to ask some other people to test on more restrictive networks though.

For whatever reason, websites are having a much harder time loading now (this problem was limited before). It can take up to 15 seconds just to see some content on the page, and sometimes even longer (similar to https://github.com/Jigsaw-Code/outline-server/issues/426).

**Comment by sbruens** [comment]
Looks like you were able to resolve your original issue. Good to know clearing iptables worked.

I'm going to close this as resolved, feel free to add any details you have to #426 if that's what you're running into.
